src/app/modules/game/services/game-view.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from "@angular/core";
+import { BehaviorSubject} from "rxjs";
+
+@Injectable({
+  providedIn: "root",
+})
+export class GameViewService {
+  private readonly _currentView = new BehaviorSubject<string>('');
```

Comment:
Why BehaviorSubject? If you neen an observable with single stream, then create an obsevable, or if you don't need initializing value, use still can use Subject

Reply:
BS stores the latest value emitted to its consumers, and whenever a new Observer subscribes, it will immediately receive the "current value" from the BehaviorSubject.

Also, initially, I planned that this service would handle all view-switching logic without components needing to be that explicit. E.g., `viewService.nextView()` would be all that is called in a component